e2e-tests/development-runtime/cypress/support/commands.js

```diff
@@ -113,3 +115,14 @@ Cypress.Commands.add(`getFastRefreshOverlay`, () => (
 Cypress.Commands.add(`assertNoFastRefreshOverlay`, () => (
   cy.get('gatsby-fast-refresh').should('not.exist')
 ))
+
+addMatchImageSnapshotCommand({
+  customDiffDir: `/__diff_output__`,
+  customDiffConfig: {
+    threshold: 0.1,
+  },
+  failureThreshold: 0.08,
+  failureThresholdType: `percent`,
+})
+
+require("@cypress/snapshot").register()
```

Comment:
Do we actually use `@cypress/snapshot`?

I think this started to cause failing dev runtime tests ( example in https://dashboard.cypress.io/projects/s3j3qj/runs/18459/test-results/b35f5663-423c-4ee2-8228-daf515b9918c ). It mentions `cy.writeFile` which we don't use anywhere, but I see that it is being used in that package ( https://github.com/cypress-io/snapshot/blob/2678cd27cbb5e8509efb8ebc100cccb49c847a37/src/index.js#L92 )

It probably means that our code there is a bit wrong, but it wasn't causing conflicts before.

Reply:
Ahhh good call. I'll remove it and see if the tests still pass. I was seeing it as a dependency within our other snapshot testing setups, but may have been mistaken that it was needed here.